cmd/node/node.go

```diff
@@ -814,12 +814,45 @@ func (app *App) listenToUpdates(ctx context.Context, appErr chan error) {
 	})
 }
 
-func (app *App) startServices(ctx context.Context, appErr chan error) error {
-	if err := app.fetcher.Start(); err != nil {
-		return fmt.Errorf("failed to start fetcher: %w", err)
+func (app *App) waitConnectedPeers(ctx context.Context) error {
+	want := uint64(app.Config.P2P.MinPeers)
+	ready := func() bool {
+		return app.host.PeerCount() >= want
+	}
+	timeout := time.Now().Add(10 * time.Minute)
+	wait := 10 * time.Second
+	for {
+		if ready() {
+			app.log.With().Info("peers ready",
+				log.Uint64("count", app.host.PeerCount()),
+				log.Uint64("want", want),
+			)
+			return nil
+		}
+		app.log.With().Warning("peers not ready",
```

Comment:
this should be debug if necessary. there are appropriate metrics that allow to see what is the actual number of peers